### ISSUE TITLE
kontena registry create  "ERROR"

### ISSUE THREAD

**Issue by santhoshkumarhirekerur** [kontena/kontena#2246]
I am running Kontena-cli on windows with tool  "Cmdr". I am able create grid deploy static images. 

Now I am trying to build instant docker image. I did kontena.yml configuration as below.  Its internally using docker-compose.yml file. docker-compose.yml file using dockerfile to build.  

because "image" is mandatory, without this I cant create custom image for Kontena. 

```yaml
stack: kontena/webhook
version: 0.3.0
services:
    webhook:
      extends:
        file: docker-compose.yml
        service: webhook
      image: registry.kontena.local/webhook:latest
      instances: 2
```

when I execute kontena registry create getting error. 

```console
λ **kontena registry create**                                                                                                                                                        
Running #<Kontena::MainCommand:0x000000037c5ad0> -- callback matcher = 'nil'                                                                                                     
Running #<Kontena::Cli::RegistryCommand:0x000000037da0e8> -- callback matcher = 'nil'                                                                                            
Running #<Kontena::Cli::Registry::CreateCommand:0x0000000385e028> -- callback matcher = 'registry create'                                                                        
D, [2017-05-03T15:24:38.478720 #8144] DEBUG -- CONFIG: Loading configuration from C:/Users/skumar/.kontena_client.json                                                           
D, [2017-05-03T15:24:38.479720 #8144] DEBUG -- CONFIG: Configuration loaded with 2 servers.                                                                                      
D, [2017-05-03T15:24:38.479720 #8144] DEBUG -- CONFIG: Current master: AUS-CBR-M1                                                                                                
D, [2017-05-03T15:24:38.479720 #8144] DEBUG -- CONFIG: Current grid: cbr-dev                                                                                                     
D, [2017-05-03T15:24:38.488720 #8144] DEBUG -- CLIENT: Excon opts: {:omit_default_port=>true, :connect_timeout=>5, :read_timeout=>30, :write_timeout=>5, :ssl_verify_peer=>false,
ntena::DebugInstrumentor}                                                                                                                                                        
[API Client Request]: GET https://10.61.40.100/v1/services/cbr-dev/registry (ssl_verify: false)  | Headers: {Accept: application/json, Authorization: Bearer}                    
[API Client Error]:                                                                                                                                                              
D, [2017-05-03T15:24:40.083720 #8144] DEBUG -- CLIENT: Request GET /v1/services/cbr-dev/registry: 404 Not Found:                                                                 
D, [2017-05-03T15:24:40.084720 #8144] DEBUG -- CLIENT: JSON parse exception: JSON::ParserError : A JSON text must at least contain two octets!                                   
[API Client Request]: GET https://IP/v1/grids/cbr-dev/nodes (ssl_verify: false)  | Headers: {Accept: application/json, Authorization: Bearer}                          
[API Client Response]: Headers: {Content-Type: application/json}  | Status: 200 | Body: "{\"nodes\":[{\"id\":\"PHUE:EUZD:MD52:T3YO:FNOJ:YE4M:VY2W:MA52:IP23:5RNJ:GDO4:SH..."     
[API Client Request]: GET https://IP/v1/secrets/cbr-dev/REGISTRY_AUTH_PASSWORD (ssl_verify: false)  | Headers: {Accept: application/json, Authorization: Bearer}       
[API Client Error]:                                                                                                                                                              
D, [2017-05-03T15:24:40.444720 #8144] DEBUG -- CLIENT: Request GET /v1/secrets/cbr-dev/REGISTRY_AUTH_PASSWORD: 404 Not Found: {"error":"Not found"}                              
[API Client Request]: GET https://IP/v1/secrets/cbr-dev/REGISTRY_HTTP_TLS_CERTIFICATE (ssl_verify: false)  | Headers: {Accept: application/json, Authorization: Bearer}
[API Client Error]:                                                                                                                                                              
D, [2017-05-03T15:24:40.594720 #8144] DEBUG -- CLIENT: Request GET /v1/secrets/cbr-dev/REGISTRY_HTTP_TLS_CERTIFICATE: 404 Not Found: {"error":"Not found"}                       
C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/kontena-cli-1.2.1/lib/kontena/cli/registry/create_command.rb:88:in `execute': uninitialized constant Kontena::Cli::Registry::CreateCommand
meError)                                                                                                                                                                         
Did you mean?  SecurityError                                                                                                                                                     
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/kontena-cli-1.2.1/lib/kontena/command.rb:200:in `run'                                                                        
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/clamp-1.1.2/lib/clamp/subcommand/execution.rb:11:in `execute'                                                                
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/kontena-cli-1.2.1/lib/kontena/command.rb:200:in `run'                                                                        
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/clamp-1.1.2/lib/clamp/subcommand/execution.rb:11:in `execute'                                                                
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/kontena-cli-1.2.1/lib/kontena/command.rb:200:in `run'                                                                        
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/clamp-1.1.2/lib/clamp/command.rb:132:in `run'                                                                                
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/kontena-cli-1.2.1/bin/kontena:17:in `<top (required)>'                                                                       
        from C:/Ruby23-x64/bin/kontena:22:in `load'                                                                                                                              
        from C:/Ruby23-x64/bin/kontena:22:in `<main>' 
```

**Comment by kke** [comment]
This is fixed in #2240

**Comment by SpComb** [comment]
Will be fixed in the 1.2.2 release.
